src/components/ImageView/Toolbar/ToolbarComponent.tsx

```diff
@@ -193,8 +193,20 @@ export class ToolbarComponent extends React.Component<ToolbarComponentProps> {
             </Menu>
         );
 
+        const baseFrame = this.props.frame;
```

Comment:
We may need to consider spatially matched frames: `this.props.frame.spatialReference ?? this.props.frame`